modules/BTBidAdapter.js

```diff
@@ -32,8 +38,17 @@ function imp(buildImp, bidRequest, context) {
   const { params, ortb2Imp } = bidRequest;
 
   if (params) {
-    const { siteId, ab, ...btBidParams } = params;
+    const { blockthrough, ...btBidParams } = params;
+
     Object.assign(imp, { ext: btBidParams });
+    if (blockthrough.auctionID) {
+      deepSetValue(
+        imp,
+        'ext.prebid.blockthrough.auctionID',
+        blockthrough.auctionID
+      );
+      delete blockthrough.auctionID;
```

Comment:
Just for my information, could you clarify why we have to delete `auctionID` here? 🙇🏻

Reply:
it was to not send it under `site.ext`